### ISSUE TITLE
NTLM Support

### ISSUE THREAD

**Issue by fancidev** [brandond/requests-negotiate-sspi#3]
If the server is configured with NTLM only, the first response will be WWW-Authenticate: NTLM, and this is not handled by the module because it recognizes "negotiate" only. But this is a pity because the code is already there to handle NTLM. Suggest refactor the code to pass "_package" as a parameter to _retry_using_http_Negotiate_auth and do something like
```
    def _response_hook(self, r, **kwargs):
        if r.status_code == 401:
            if 'negotiate' in r.headers.get('WWW-Authenticate', '').lower():
                return self._retry_using_http_Negotiate_auth(r, 'Negotiate', kwargs)
            if 'ntlm' in r.headers.get('WWW-Authenticate', '').lower():
                return self._retry_using_http_Negotiate_auth(r, 'NTLM', kwargs)
```
Then it will work.

Thanks for the great module.

**Comment by brandond** [comment]
Since Negotiate handles switching between Kerberos and NTLM under the hood, a similar approach might also work for servers that only advertise Kerberos support, but I don't have an easy way to test it.

**Comment by brandond** [comment]
Just added explicit NTLM support in https://github.com/brandond/requests-negotiate-sspi/commit/1535683e621805c54f0c8910ecd8c17c713a2660 and released as 0.3.0, want to give it a try?

**Comment by fancidev** [comment]
Tried and works! Thanks!
